encrypted-media-respec.html

```diff
@@ -3165,7 +3165,21 @@
         <p>The use of identifiers, especially <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a>, by implementations presents a privacy concern.
           This section defines requirements for avoiding or at least mitigating such concerns.
         </p>
-        <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/117">Issue 117</a> - This section should apply to all persistent identifiers, not just <a def-id="distinctive-identifiers"></a>.</p>
+        <div class="note">
+          <p>In summary:</p>
+          <ul>
+            <li><p><a href="#limit-or-avoid-use-of-distinctive-identifiers-and-permanent-identifiers">Limit or Avoid use of Distinctive Identifiers and Permanent Identifiers</a>.</p></li>
+            <li>
+              <p>
+                All identifers that are not <a href="#permanent-identifier">Permanent Identifiers</a> MUST be <a href="#per-origin-identifiers">unique per-origin</a> and <a href="#allow-identifiers-cleared">clearable</a>.</p>
+                They SHOULD be <a href="#encrypt-identifiers">encrypted</a> when exposed outside the client.
+              </p>
+            </li>
+            <li><p><a def-id="distinctive-identifiers"></a> MUST be <a href="#encrypt-identifiers">encrypted</a> when exposed outside the client, <a href="#per-origin-identifiers">unique per-origin</a>, and <a href="#allow-identifiers-cleared">clearable</a>.</p></li>
```

Comment:
Yes, there is duplication here. My goal was to have a single place to look and see the requirements for various types of identifiers without having to put a bunch of categories together. That information exists normatively in the sections below - this is intended to help the reader.